pymc3/step_methods/hmc/base_hmc.py

```diff
@@ -198,6 +198,10 @@ def astep(self, q0):
 
         return hmc_step.end.q, [stats]
 
+    def reset_tuning(self, start=None):
```

Comment:
@dfm sorry for the retroactive review, but what is the purpose of this function? It doesn't seem to be used anywhere in the PyMC3 code.

cc https://github.com/eigenfoo/littlemcmc/pull/68

Reply:
Woops sorry, just dug around a bit deeper. I see now!